experimental/devices/mfrc522/mfrc522.go

```diff
@@ -188,6 +208,15 @@ func (r *Dev) SetAntenna(state bool) error {
 	return r.clearBitmask(commands.TxControlReg, 0x03)
 }
 
+// SetAntennaGain configures antenna signal strength.
+//
+//	gain - signal strength from 0 to 7.
+func (r *Dev) SetAntennaGain(gain int) {
+	if 0 <= gain && gain <= 7 {
```

Comment:
I think you should to protect it via the mutex then, since I suspect the caller may be in a separate goroutine.

Reply:
I don't think it can actually hurt but added a mutex.